StreamChatUI-XCFrameworks.podspec

```diff
@@ -0,0 +1,26 @@
+Pod::Spec.new do |spec|
+  spec.name = "StreamChatUI-XCFrameworks"
+  spec.version = "4.5.0"
+  spec.summary = "StreamChat UI Components"
+  spec.description = "StreamChatUI SDK offers flexible UI components able to display data provided by StreamChat SDK."
+
+  spec.homepage = "https://getstream.io/chat/"
+  spec.license = { :type => "BSD-3", :file => "LICENSE" }
+  spec.author = { "getstream.io" => "support@getstream.io" }
+  spec.social_media_url = "https://getstream.io"
+
+  spec.swift_version = "5.2"
```

Comment:
xcframeworks will be built by Xcode 13.0 command line tools, let's reflect the correct Swift version here

Reply:
True that!